From d7337cad4dbe6caf9535d3539daed353dd425dc1 Mon Sep 17 00:00:00 2001
From: mingzhu wang <mingzhu.wang@transsion.com>
Date: Tue, 9 Jun 2026 02:15:00 +0000
Subject: [PATCH 00942/15333] kernel: exit: fix coding style missing spaces

Add spaces around bitwise AND and shift operators in sys_exit()
to comply with the Linux kernel coding style.

Signed-off-by: mingzhu wang <mingzhu.wang@transsion.com>
Link: https://patch.msgid.link/20260609021436.1739-1-mingzhu.wang@transsion.com
Signed-off-by: Christian Brauner (Amutable) <brauner@kernel.org>
---
 kernel/exit.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/exit.c b/kernel/exit.c
index 1056422bc101..7ac52196d819 100644
--- a/kernel/exit.c
+++ b/kernel/exit.c
@@ -1111,7 +1111,7 @@ void __noreturn make_task_dead(int signr)
 
 SYSCALL_DEFINE1(exit, int, error_code)
 {
-	do_exit((error_code&0xff)<<8);
+	do_exit((error_code & 0xff) << 8);
 }
 
 /*
-- 
2.55.0

